sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/channel/SessionTracker.java

```diff
@@ -162,20 +121,8 @@ private void sendStartSessionIfNeeded() {
             /* New session: generate a new identifier. */
             mSid = UUIDUtils.randomUUID();
```

Comment:
Might this be done by `sendStartSessionIfNeeded`?

Reply:
that code is already inside `sendStartSessionIfNeeded`, not sure what you meant